src/digitalIdentity-backend/src/main/java/didentity/amos/digitalIdentity/controller/AuthenticationController.java

```diff
@@ -67,4 +67,50 @@ public class AuthenticationController {
         return "\"password and username do not match\"";
 
     }
+
+
+    @PostMapping(path="/update")
+    public @ResponseBody String update(@RequestParam String name, @RequestParam String surname, @RequestParam String birthday,
+    @RequestParam String email, @RequestParam String company, @RequestParam String team, @RequestParam String user_role) {
+        Iterable<User> users = userRepository.findAll();
+        
+        // TODO Jannik: findAll() ist ziemlich inperformant; Ich wusste leider nicht wie man e_mail und password direkt im Repository abfragen kann
```

Comment:
For the future: User .stream().filter() and do not use the traditional for loop.